docs/UserGuide.md

```diff
@@ -176,7 +176,7 @@ Edits the specified fields of an existing person in Dormie.
 - Edit function will replace the specified field with the new input
 - Edits the person at the specified INDEX. The index refers to the index number shown in the displayed person list. The index must be a positive integer. eg. 1, 2, 3
 - Existing values will be updated to the input values.
-- Minimum 1 field must be specified. Eg. `n/NAME` is a field.
+- Minimum 1 field must be specified. eg. `n/NAME` is a field.
```

Comment:
Should it be "E.g." instead? Since we use a period before it, it should be the start of a new sentence.

Reply:
I standardized it to just eg. for all of it ah, looks more cohesive I guess haha